tests/Feature/testdata/events_request_body.golden

```diff
@@ -0,0 +1 @@
+{"events":[{"channel":"test-channel","name":"test-event","data":"{\"test-key\":\"test-val\"}"},{"channel":"test-channel2","name":"test-event2","data":"{\"test-key\":\"test-val2\"}"}]}
```

Comment:
```suggestion
[{"channel":"test-channel","name":"test-event","data":"{\"test-key\":\"test-val\"}"},{"channel":"test-channel2","name":"test-event2","data":"{\"test-key\":\"test-val2\"}"}]
```

According to https://docs.broadcastt.xyz/references-rest-api/#post-events

> Request data must be an array of events.

I checked the code of the API handler at it seems we expect a JSON array and not a JSON object which wraps the JSON array. It seems it was implemented incorrectly. I would suggest to fix this in this PR.